ros2/cc_defs.bzl

```diff
@@ -75,7 +75,7 @@ def _ros2_cpp_exec(target, name, ros2_package_name, set_up_ament, idl_deps, **kw
 
     launcher_target_kwargs, binary_kwargs = split_kwargs(**kwargs)
     target_impl = name + "_impl"
-    _ros2_cc_target(cc_binary, "cpp", target_impl, ros2_package_name, tags = ["manual"], **binary_kwargs)
+    _ros2_cc_target(cc_binary, "cpp", target_impl, ros2_package_name, tags = launcher_target_kwargs.get("tags"), **binary_kwargs)
```

Comment:
What's the reasoning here?

Reply:
We use tags for internal tooling (e.g. doing some build graph analysis and restrictions). The rule was throwing away the tags and unconditionally replacing it with `manual`. This change keeps the tags for the cc_binary target. The manual one in L89 is still there.

Reply:
Hm, internal targets are typically marked as manual. The top level target should receive the tags passed through macro.  Do you need this for e.g. some code analysis tooling?

Reply:
Yup, e.g. clang-tidy aspect integration.

Reply:
I get your intention. Though, IMO, this is a very custom setup. FWIK, internal targets are best to be tagged as manual. Instead of relying on tags, you can query for target kind and then run analyzers on such targets.

Reply:
that's not really the same -- for custom tools you in many cases surely you could go with queries instead. But some existing tools like https://github.com/erenon/bazel_clang_tidy rely on tags at this time.

Reply:
You refer to this thing https://github.com/erenon/bazel_clang_tidy/blob/master/example/BUILD#L17, right? I'm fine to set `launcher_target_kwargs.get("tags") + ["manual"]`. Does that work for you?

Reply:
sgtm, updated.